mailjet.php

```diff
@@ -175,8 +175,7 @@ private function initContext()
 
     public function install()
     {
-        //$this->account = array(); // **
-        $this->account = ($account = Tools::jsonDecode(Configuration::get('MAILJET'))) ? $account : $this->account;
+        $this->account = ($account = json_decode(Configuration::get('MAILJET'))) ? $account : $this->account;
         $this->account->TOKEN = Tools::getValue('token');
```

Comment:
Hello @oleksandr-mykhailenko FYI this token is never set in 1.7.x because it's not `token` but `_token` in this version.

Regards